AODVv2-pandoc-edits/AODVv2.mkd

```diff
@@ -1798,62 +1798,73 @@ MAX_SEQNUM_LIFETIME MUST be configured to have the same values for all AODVv2 ro
-*  RREP_RETRIES: Routers with lower values are more likely to blacklist neighbors when there is a temporary fluctuation in link quality.
-*  MAX_METRIC[MetricType]: No interoperability problems due to variations on different routers, but
-   routers with lower values may exhibit overly restrictive behavior during route comparisons.
-*  MAX_HOPCOUNT: Routers with a value too small would not be able to discover routes to distant addresses.
-*  INFINITY_TIME: No interoperability problems due to variations on different routers, but if a
-   lower value is used, route state management may exhibit overly restrictive behavior.
+   <!-- Need to figure out what the time format. -->
 
 ## Local Settings {#other}
-The following table lists AODVv2 parameters which SHOULD be administratively configured for each router:
 
-Name                      Default Value             Description
----                       ------------              --------------
-InterfaceSet                                        [](#interfaceslist)
-Router Client Set                                   [](#clients)
-BUFFER_SIZE_PACKETS       2                         [](#route_discovery)
-BUFFER_SIZE_BYTES         MAX_PACKET_SIZE [TBD]     [](#route_discovery)
-CONTROL_TRAFFIC_LIMIT     [TBD - 50 pkts/sec?]      [](#aodv_msgs)
+The following lists shows AODVv2 parameters which SHOULD be administratively configured for each router:
+
+InterfaceSet
+:   <vspace/>The list of interfaces which AODVv2 can use for participating in route discovery and maintenance. Further 
+details can be found in [](#interfaceslist).
+
+Router Client Set
+:   <vspace/>The set of addresses for which AODVv2 is configured to perform route discovery, as defined in [](#clients).
+
+CONTROL_TRAFFIC_LIMIT
+:   <vspace/>The limit on the amount of control traffic that can be generated by an AODVv2 router. This value should be 
+based on the average size of a packet containing an AODVv2 message and the bandwidth available. The limit should be 
+calculated so that AODVv2 traffic does not exceed a certain percentage of the bandwidth. The use of CONTROL_TRAFFIC_LIMIT 
+is described in [](#aodv_msgs). 
+<!-- [TBD - 50 pkts/sec?] TODO Charlie to suggest a value -->
 
-^[admincontrol::Configuration for Local Settings]
+
+<!-- BUFFER_SIZE_PACKETS       2                         [](#route_discovery)
+BUFFER_SIZE_BYTES         MAX_PACKET_SIZE [TBD]     [](#route_discovery)
+-->
 
 ## Network-Wide Settings
+
 The following administrative controls MAY be used to change the operation of the network. The same
 settings SHOULD be used across the network. Inconsistent settings at different routers in the network
 will not result in protocol errors, but poor performance may result.
 
-Name                       Default               Description
-------                     --------              ----------
-ENABLE_IDLE_IN_RERR        Disabled              [](#RERR_gen)
+ENABLE_IDLE_IN_RERR
```

Comment:
If I don't get outvoted, it's staying the same ;) Just as a reminder, I updated a few of the tables in the Configuration section so that they look the same as the terminology section and allow a bigger description of the parameter.